examples/flutter_gallery/lib/demo/contacts_demo.dart

```diff
@@ -54,7 +52,8 @@ class _ContactItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final ThemeData themeData = Theme.of(context);
-    final List<Widget> columnChildren = lines.sublist(0, lines.length - 1).map<Widget>((String line) => Text(line)).toList();
+    final List<Widget> columnChildren =
+        lines.sublist(0, lines.length - 1).map<Widget>((String line) => Text(line)).toList();
```

Comment:
this is no better than the original imho

Reply:
Trailing comment to the rescue ;)

```dart
    final List<Widget> columnChildren = lines //
        .sublist(0, lines.length - 1)
        .map<Widget>((String line) => Text(line))
        .toList();
```